### PR TITLE
Minor dependency updates

### PR DESCRIPTION
**Pull request by craigcomstock**
- **Updated openssl from 3.3.2 to 3.4.0**
- **Updated php from 8.3.12 to 8.3.13**
with https://github.com/cfengine/system-testing/pull/579

### CODE REVIEW

**Comment by craigcomstock** [comment]
@cf-bottom jenkins

**Comment by craigcomstock** [comment]
ubu22 and mingw failures are lack of removing the 0007 windows patch which I did in the 3.24.x PR for this same upgrade of openssl. Will add that change here.

one host failed during bootstrap but it is hard to parse which one, ENT-12418 to fix that.

sequential-tests timed out during an agent run step.

Will wait for ENT-12416 ssh stability improvement before another jenkins run.

**Comment by cf-bottom** [comment]
Sure, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/11337/)

**Jenkins:** https://ci.cfengine.com/job/pr-pipeline/11337/

**Packages:** http://buildcache.cfengine.com/packages/testing-pr/jenkins-pr-pipeline-11337/

**Comment by craigcomstock** [comment]
So far the x86 mingw agent seems to work fine on two instances I am running it on.

I will check CI failures and merge if I can, then proceed to LTS deps updates PRS and come back here to update the deps table after all are merged.

**Comment by craigcomstock** [comment]
@cf-bottom jenkins

**Comment by cf-bottom** [comment]
Sure, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/11339/)

**Jenkins:** https://ci.cfengine.com/job/pr-pipeline/11339/

**Packages:** http://buildcache.cfengine.com/packages/testing-pr/jenkins-pr-pipeline-11339/

**Comment by craigcomstock** [comment]
@cf-bottom jenkins please

**Comment by cf-bottom** [comment]
Alright, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/11356/)

**Jenkins:** https://ci.cfengine.com/job/pr-pipeline/11356/

**Packages:** http://buildcache.cfengine.com/packages/testing-pr/jenkins-pr-pipeline-11356/

**Comment by craigcomstock** [comment]
green enough: [Build Status](https://ci.cfengine.com/job/sequential-tests/7816/console)

**Review by larsewi — APPROVED** [review]
